assets/js/bootstrap-migration/common-components/loader.tpl.html.liquid

```diff
@@ -0,0 +1,5 @@
+<div ng-if="$ctrl.isLoading" class="loader bg-dark-transparent position-fixed t-0 b-0 l-0 r-0 d-flex align-items-center justify-content-center">
```

Comment:
This code will never work because $ctrl is not built-in variable, we declare it in controller as `var $ctrl = this;`

Reply:
(also a tip: instead of `t-0 b-0 l-0 r-0` you may use `t-0 w-100 h-100` if you like it more; doesn't make sense for me, you may leave as is)

Reply:
> (also a tip: instead of `t-0 b-0 l-0 r-0` you may use `t-0 w-100 h-100` if you like it more; doesn't make sense for me, you may leave as is)

i think i'll leave it as is, maybe it will be helpful in future

Reply:
> This code will never work because $ctrl is not built-in variable, we declare it in controller as `var $ctrl = this;`
My bad, 🤦‍♂️ .  Done